### DISCUSSION TITLE
everyone-should-have-a-pikvm-in-their-desk-area/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#232] [Announcements]
# everyone-should-have-a-pikvm-in-their-desk-area/

What do you do when you want to tinker with a bare metal system? For years, I'd go drag an old monitor, keyboard, and mouse out of a cupboard. Then, I'd inevitably spend 20 minutes looking for an ethernet cable that was the right length before finding that I had

https://blog.ktz.me/everyone-should-have-a-pikvm-in-their-desk-area/

**Comment by LinuxTeamster** [comment]
Great article, thanks for sharing
